packages/allow-scripts/src/types/common.ts

```diff
@@ -42,4 +56,60 @@ declare global {
      excessPolicies: string[];
      somePoliciesAreMissing: boolean;
   }
+
```

Comment:
if any of the below are user-accessible via a function signature (referenced directly or otherwise), they should be exported